packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -305,23 +295,29 @@ A custom email address allows you to present a branded experience to your end us
 
 Okta sends your super admins a confirmation email after your custom email address is configured and operating correctly. To ensure continuous operation, Okta polls your custom email domain once every 24 hours. If a problem occurs, Okta alerts super admins by email, and Okta-generated emails are sent from the default address `noreply@okta.com` until the problem is resolved.
 
-## Configure a custom email address
+### Use the Brands API
+
+The [Email Domains API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/EmailDomain/) is a feature that allows you to create, update, and delete email domains for your organization.
+
+### Configure a custom email address
+
+1. In the Admin Console, go to **Customizations** > **Brands**, and then select the brand you want.
 
-1. In the Admin Console, go to **Customizations** > **Emails**. (If you enabled multibrand customization, go to **Customizations** > **Brands**, and then select the brand you want. In the **Domains** tab > **Email** section, click **Add domain**).
+2. In the **Domains** tab > **Emails** section, click **Add domain**.
 
-2. In the **Email address** field, enter the email address that you want to send the system notification emails from. This email address appears in the emails sent to your users.
+3. In the **Email address** field, enter the email address that you want to send the system notification emails from. This email address appears in the emails sent to your users.
 
-3. In the **Name of email sender** field, enter the name of sender. This name appears as the sender in the emails sent to your users.
+4. In the **Name of email sender** field, enter the name of sender. This name appears as the sender in the emails sent to your users.
```

Comment:
trying to make this less "name of the sender" "email sender" "sender of the emails"
```suggestion
4. Enter the sender name in the **Name of email sender** field. This name appears as the sender of the emails sent to your users.
```